services/validatorWorker/follower.js

```diff
@@ -43,19 +44,26 @@ function tick(adapter, channel) {
 function onNewState(adapter, {channel, balances, newMsg, approveMsg}) {
 	const prevBalances = toBNMap(approveMsg ? approveMsg.balances : {})
 	const newBalances = toBNMap(newMsg.balances)
+	const { balancesAfterFees } = newMsg
 
 	if (!isValidTransition(channel, prevBalances, newBalances)) {
 		console.error(`validatatorWorker: ${channel.id}: invalid transition requested in NewState`, prevBalances, newBalances)
 		return { nothingNew: true }
 	}
 
+	if(!isValidValidatorFees(channel, newBalances, balancesAfterFees)) {
```

Comment:
this approach is correct, although I have a few thoughts

1. `balancesAfterFees` and `newBalances` should be in the same format (BN map) even before they come to `isValidValidatorFees`)
2. I think from a game theoretical standpoint this validation is not very useful and can be dropped; the reason is that the leader decides how the balances get distributed anyway (since it's their money); the follower's job is to ensure the leader can't take money back (OUTPACE rules); 


i don't want to do anything about (2) right now, but I want you to think about it